src/Aspire.Dashboard/Components/Controls/MetricTable.razor

```diff
@@ -0,0 +1,82 @@
+﻿@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Otlp.Model.MetricValues
+@using Aspire.Dashboard.Resources
+@namespace Aspire.Dashboard.Components.Controls
+@inject IStringLocalizer<ControlsStrings> Loc
+
+<div id="metric-table-container" style="max-height: 60vh; overflow-y: scroll; margin-bottom: 10px;">
+    <FluentDataGrid Items="@FilteredMetrics.AsQueryable()" ItemSize="35" Virtualize="false">
```

Comment:
Why is virtualize false? There is a noticeable delay when switching between 10 latest rows and all rows. The delay is from adding all the new rows to the table. That's not a problem with virtualization.

Reply:
Blazor errors if the duration is set to 30 minutes or greater, apps have been running that long, and you switch to view all data.

Virtualization is definitely required.

Reply:
If we flip the order so that new data points are added to the top of the table instead of the bottom, re-adding virtualization is easy. The current approach for reading to screen readers is to obtain the text content of each cell in the latest row, which is not possible if the row is virtualized away. 

I'll change the order, which will take care of this & your previous comment.